www/css/styles.css.pp

```diff
@@ -505,6 +505,10 @@
   transition: color 0.2s;
 }
 
+langwww {
+  color: rgb(115,115,115);
+  font-weight: bold;
+}
 
```

Comment:
I don't understand CSS that well, but I would have thought that this needs a `.` in front of it?

Also, `langwww` has an opacity of .5 below which is what makes it dark. Maybe it would work better to change that instead?

Reply:
@rfindler actually, this is "correct" in a sense that there's really a tag named `langwww`. So this CSS is indeed correct. But what should really happen is that Pollen shouldn't generate `langwww`. Instead, it should be a `div` with a class `langwww`.

Reply:
To elaborate, the fix is to include 

```
◊(require pollen/tag)
◊(define langwww (default-tag-function 'div #:class "langwww"))
```

in either `www/index.html.pm` or `www/util.rkt`.

Reply:
Thanks!

Reply:
What sorawee said. I wasn't sure if the `langwww` tag was really intended since it's not a standard HTML tag but technically it's possible to extend HTML with additional tags...

There is a `.langwww` CSS rule present in styles.css.pp which applies to `langwww` class and it's not being used at all.

Reply:
Okay, I didn't like the boldface everywhere so I went with a darker color. [What do you think?](http://www.cs.northwestern.edu/~robby/tmp/Web/www)

Reply:
It looks better already, but would you mind changing the rgb values to `rgb(100,100,100)` pretty please? :-)

Reply:
Done!